### PR TITLE
LWDEV-8902 Queue logger should log all queue sizes

### PR DESCRIPTION
**Pull request by rocketmannot**

### CODE REVIEW

File: src/test/kotlin/com/lykke/matching/engine/services/StopLimitOrderTest.kt [hunk 1]
```diff
@@ -78,6 +78,7 @@ class StopLimitOrderTest : AbstractTest() {
         testBalanceHolderWrapper.updateReservedBalance("Client1", "USD", 0.0)
 
         testDictionariesDatabaseAccessor.addAssetPair(AssetPair("BTCUSD", "BTC", "USD", 6))
+        testDictionariesDatabaseAccessor.addAssetPair(AssetPair("BTCEUR", "BTC", "USD", 6, maxValue = BigDecimal.valueOf(8000)))
```
Comment (barsukovskii) [inline src/test/kotlin/com/lykke/matching/engine/services/StopLimitOrderTest.kt]: Looks like error of merging. This change is not related to the task

File: src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt [hunk 2]
```diff
@@ -152,4 +188,25 @@ open class DatabaseAccessorConfig {
...
+
+    //<editor-fold desc="Persist listeners>
+    @Bean
+    open fun persistedWalletsApplicationEventPublisher(updatedWalletsQueue: BlockingQueue<Collection<Wallet>>,
+                                                       listeners: Optional<List<QueueConsumer<Collection<Wallet>>?>>): SimpleApplicationEventPublisher<Collection<Wallet>> {
+        return ApplicationEventPublisherImpl(updatedWalletsQueue, listeners)
+    }
+
+    @Bean
+    open fun persistedOrdersApplicationEventPublisher(updatedStopOrderBooksQueue: BlockingQueue<Collection<OrderBookPersistenceData>>,
```
[the remark this answers is not in the document]
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt]: renamed persistedOrdersApplicationEventPublisher -> persistedStopOrdersApplicationEventPublisher

File: src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt [hunk 3]
```diff
@@ -152,4 +188,25 @@ open class DatabaseAccessorConfig {
...
+                                                      listeners: Optional<List<QueueConsumer<Collection<OrderBookPersistenceData>>?>>): SimpleApplicationEventPublisher<Collection<OrderBookPersistenceData>> {
```
[the remark this answers is not in the document]
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt]: changed, now persistence data has wrapper like in rabbit mq messages

File: src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt [hunk 4]
```diff
@@ -152,4 +188,25 @@ open class DatabaseAccessorConfig {
...
+        return ApplicationEventPublisherImpl(updatedStopOrderBooksQueue, listeners)
+    }
+
+    @Bean
+    open fun persistedStopApplicationEventPublisher(updatedOrderBooksQueue: BlockingQueue<Collection<OrderBookPersistenceData>>,
```
[the remark this answers is not in the document]
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt]: fixed

[the remark this answers is not in the document]
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/config/spring/DatabaseAccessorConfig.kt]: fixed

File: src/main/kotlin/com/lykke/matching/engine/database/reconciliation/OrdersReconciliationService.kt [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+import com.lykke.matching.engine.holders.StopOrdersDatabaseAccessorsHolder
+import com.lykke.matching.engine.utils.config.Config
+import org.apache.log4j.Logger
+import org.springframework.boot.ApplicationArguments
+import org.springframework.boot.ApplicationRunner
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+
+@Component
+@Order(1)
```
[the remark this answers is not in the document]
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/database/reconciliation/OrdersReconciliationService.kt]: as far as i understand it does not matter what migration called first accounts or orders, from spring perspective components can have the same order
